### PR TITLE
Runner module

### PR DESCRIPTION
**Pull request by toffernator**
## Describe your changes

Created `runner.py` which is responsible for taking a quality model, executing its metrics and aggregations functions, and producing a .csv file. `runner.py` includes the date/time down to the second as the tile of the .csv file and adds the name of the quality model used to produce the values in the .csv file.

We should merge #34 first, and use that here to skip the qualitymodel=<qualtiymodel> line that is now written to csv.

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [x] Have I requested and notified the boys for review of the pull-request?

### CODE REVIEW

**Review by FrederikRothe — COMMENTED** [review]
A couple of clarifying comments on the code, maybe just me being slow 🐌

File: models/code_climate.py
Comment (toffernator) [reply models/code_climate.py:60]: As I understand CodeClimate, they approximate cyclomatic complexity with return statements, and argue that it is good enough when paired with cognitive complexity.

https://docs.codeclimate.com/docs/cyclomatic-complexity#cyclomatic-vs-cognitive-complexity

File: byoqm/runner.py
Comment (toffernator) [reply byoqm/runner.py:22]: will update

Comment (SadCheeto) [reply byoqm/runner.py]: changed it to a path

File: byoqm/main.py
Comment (Kobo-coder) [inline byoqm/main.py:27]: is this the final decision as to how users changes models, or do we later move this to a parameter for a CLI command? Doesn't stop from merging.

File: byoqm/runner.py
Comment (Kobo-coder) [inline byoqm/runner.py:81]: Perhaps we should discuss in person, the use of `shell=True` with regards to Helge's recommendation.

File: byoqm/runner.py
Comment (Kobo-coder) [inline byoqm/runner.py:100]: I believe its possible to use pandas to write a csv file as discussed previously. Again, not something that will keep the branch from being merged. Perhaps something we could change later.

**Review by Kobo-coder — APPROVED** [review]
Left some comments. An answer to them before merging would be very appreciated.

File: byoqm/main.py
Comment (SadCheeto) [reply byoqm/main.py:27]: I Think we should discuss this tomorrow. Some of these parameters are optional, but this one should be required

File: byoqm/runner.py
Comment (SadCheeto) [reply byoqm/runner.py:100]: I don't think it is that important as of now, but we could keep it in the backlog

**Pull request merged**
